### PR TITLE
API-Docs Category

### PR DESCRIPTION
**Pull request by hungbkdn**

### CODE REVIEW

**Comment by at-vietpham** [comment]
LGTM

File: docs/api-references/categories.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+    "path": "http://link-to-host.com/api/categories",
+    "prev_page_url": null,
+    "from": 1,
+    "to": 15,
+    "data": [
+        {
+            "id": 1,
+            "name": "Dare, Beahan and Botsford",
+            "description": "Optio ad aspernatur quis. Alias quae quo ea est. Dolores laudantium tempore fugiat odio placeat.",
+            "created_at": "2017-07-31 13:58:02",
```
Comment (ghost) [inline docs/api-references/categories.md]: If don't use `created_at` and `updated_at`, you don't need to return them.

File: docs/api-references/categories.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+            "updated_at": "2017-07-31 13:58:02"
+        }
+    ],
+    "success": true
+}
+```
+
+### `GET` Food by Category
+```
+/categories/{id}
```
Comment (ghost) [inline docs/api-references/categories.md]: I think this is not a good endpoint.
`/categories/{id}` by Restful is show detail of category has ID `{id}`, don't display foods of category `{id}`
You can try to use: **GET** `/categories/{id}/foods`

File: docs/api-references/categories.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+            "description": "Eveniet corporis dolorum iste sit et enim ea. Unde est ducimus aut minus repudiandae. Ex architecto cum aut omnis est earum sunt. Voluptatibus quisquam ullam reprehenderit aliquam.",
+            "created_at": "2017-07-31 13:58:02",
+            "updated_at": "2017-07-31 13:58:02"
+        }
+    ],
+    "success": true
+}
+```
+
+### `GET` Food by Category
```
Comment (ghost) [inline docs/api-references/categories.md]: Use **Foods**, because has many foods in a category.

**Review by ghost — COMMENTED** [review]
Please check my comment

File: docs/api-references/categories.md [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+Get all informations about this category
+#### Request Header
+| Key | Value |
+|---|---|
+|Accept|application/json
+
+#### Sample Response
+```json
+{
+    "id": 12,
```
Comment (ghost) [inline docs/api-references/categories.md]: Please make `"data"` key to containing category info.

**Review by ghost — COMMENTED** [review]
Please check my comment.

File: docs/api-references/categories.md [hunk 5]
```diff
@@ -0,0 +1,232 @@
+## Category - API
+
+### `GET` List Category
+```
+/categories
```
Comment (ghost) [inline docs/api-references/categories.md]: If your route has prefix `/api/` please update your URI

**Comment by hungbkdn** [comment]
I have updated, please check it again!
Thank you so much @at-vietpham

**Comment by ghost** [comment]
LGTM 🉑

**Pull request merged**